MediaBrowser.Providers/Movies/ImdbExternalUrlProvider.cs

```diff
@@ -0,0 +1,27 @@
+using System.Collections.Generic;
+using MediaBrowser.Controller.Entities;
+using MediaBrowser.Controller.Providers;
+using MediaBrowser.Model.Entities;
+
+namespace MediaBrowser.Providers.Movies;
+
+/// <summary>
+/// External URLs for IMDb.
+/// </summary>
+public class ImdbExternalUrlProvider : IExternalUrlProvider
+{
+    /// <inheritdoc/>
+    public string Name => "IMDb";
+
+    /// <inheritdoc/>
+    public IEnumerable<string> GetExternalUrls(BaseItem item)
+    {
+        var baseUrl = "https://www.imdb.com/";
+        var externalId = item.GetProviderId(MetadataProvider.Imdb);
```

Comment:
Can't we use `TryGetProviderId` instead?
(same for all the following instances)